device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsDeviceTelemetryTest.java

```diff
@@ -29,6 +30,7 @@
 import java.util.Map;
 import java.util.UUID;
 
+import static com.microsoft.azure.sdk.iot.device.MessageProperty.OUTPUT_NAME_PROPERTY;
 import static junit.framework.TestCase.assertFalse;
 import static org.junit.Assert.*;
 
```

Comment:
Could you update coverage ? Here and everywhere.

Reply:
sure thing, updated